l1-contracts/test/Rollup.t.sol

```diff
@@ -113,17 +115,11 @@ contract RollupTest is DecoderBase {
 
     _testBlock("mixed_block_1", false);
 
-    uint256 currentSlot = rollup.getCurrentSlot();
-    (,, uint128 slot) = rollup.blocks(1);
-    uint256 prunableAt = uint256(slot) + rollup.TIMELINESS_PROVING_IN_SLOTS();
-
-    vm.expectRevert(
-      abi.encodeWithSelector(Errors.Rollup__NotReadyToPrune.selector, currentSlot, prunableAt)
-    );
+    vm.expectRevert(abi.encodeWithSelector(Errors.Rollup__NothingToPrune.selector));
     rollup.prune();
   }
 
-  function testPrune() public setUpFor("mixed_block_1") {
```

Comment:
I don't like that you are using the `testBlock` to prune. We want to see that both things works. Create a separate test if you want to test a separate thing.